src/display/text_layer.js

```diff
@@ -65,6 +65,9 @@ const ascentCache = new Map();
 let _canvasContext = null;
 
 function getCtx(lang = null) {
+  if (_canvasContext && _canvasContext.canvas.lang !== (lang || "")) {
```

Comment:
```suggestion
function getCtx(lang = "") {
  if (_canvasContext && _canvasContext.canvas.lang !== lang) {
```

Reply:
Note that the default value is `null`, from the API, so this code still needs to handle that case as well.